libvuln/jsonblob/jsonblob.go

```diff
@@ -73,8 +84,15 @@ func (l *Loader) Next() bool {
 			l.next.Fingerprint = l.de.Fingerprint
 			l.next.Date = l.de.Date
 		}
-		l.next.Vuln = append(l.next.Vuln, l.de.Vuln)
-		l.de.Vuln = nil // Needed to ensure the Decoder allocates new backing memory.
+		i := len(vs)
+		vs = append(vs, claircore.Vulnerability{})
+		if err := json.Unmarshal(l.de.Vuln.RawMessage, &vs[i]); err != nil {
```

Comment:
When is RawMessage populated?

Reply:
On this line

Reply:
Sorry, I misunderstood this. It happens in the actions in the `for` loop.